spot-ml/src/main/scala/org/apache/spot/lda/SpotLDAWrapper.scala

```diff
@@ -85,41 +80,67 @@ object SpotLDAWrapper {
         wordDictionary,
         sqlContext)
 
+    val corpusSize = ldaCorpus.count()
+
     docWordCountCache.unpersist()
 
     //Instantiate optimizer based on input
-    val optimizer = new EMLDAOptimizer
+    val ldaOptimizer = ldaOptimizerOption match {
+      case "em" => new EMLDAOptimizer
+      case "online" => new OnlineLDAOptimizer().setOptimizeDocConcentration(true).setMiniBatchFraction({
```

Comment:
where did these values come from? are they taken from spark documentation, a paper, or some experiments that we ran?